packages/@aws-cdk/aws-route53-targets/lib/api-gatewayv2-domain-name.ts

```diff
@@ -0,0 +1,16 @@
+import * as apigv2 from '@aws-cdk/aws-apigatewayv2';
+import * as route53 from '@aws-cdk/aws-route53';
+
+/**
+ * Defines an API Gateway V2 domain name as the alias target.
+ */
+export class ApiGatewayv2Domain implements route53.IAliasRecordTarget {
```

Comment:
curious: how does the casing here translate for Python? You can run `yarn package` to generate the distributions for other languages.

would it be `api_gatewayv2_domain` ? wondering if it should be `ApiGatewayV2`

Reply:
Is there a way to easily run this without installing `yarn`, `javac` and others? Like a docker image?

Reply:
`yarn package`, but class names are fine - ignore this comment!